docs/application/dotnet/guides/account/account.md

```diff
@@ -88,15 +88,15 @@ To enable your application to use the account management functionality:
 
 To create an account, set its properties, and add it to the account database:
 
-1.  Create an account using the `CreateAccount()` method of the [Tizen.Account.AccountManager.Account](https://samsung.github.io/TizenFX/latest/api/Tizen.Account.AccountManager.Account.html) class:
+1.  Create an account using the `CreateAccount()` method of the [Tizen.Account.AccountManager.Account](/application/dotnet/api/TizenFX/latest/api/Tizen.Account.AccountManager.Account.html) class:
```

Comment:
I am not able to access the links on staging, neither on Git. If the links will be resolved and shall be taken care of, kindly ignore the comments for links
https://docs1.stg.tizen.org/staging/1387/application/dotnet/guides/account/account/

Reply:
Same as above.